crates/volta-core/src/error/details.rs

```diff
@@ -816,11 +818,14 @@ Please ensure you have correct permissions.",
 {}",
                 REPORT_BUG_CTA
             ),
-            ErrorDetails::ParseHooksError => write!(
+            ErrorDetails::ParseHooksError { file } => write!(
                 f,
-                "Could not parse hooks.toml configuration file.
+                "Could not parse configuration file.
 
-Please ensure the file is correctly formatted."
+{}
```

Comment:
Nitpick: We've generally used `from {}` in other errors that reference a file name. See e.g. the message for `ErrorDetails::PackageReadError`.

Perhaps:

```rust
"Could not parse Volta configuration
from {}

Please ensure the file is correctly formatted."
```

Reply:
Done.